packages/suite/src/components/suite/modals/ReduxModal/DeviceContextModal/PassphraseWalletConfirmation.tsx

```diff
@@ -0,0 +1,205 @@
+                    </Card>
+                </Rows>
+            </>
+        );
+    }
+
+    if (contentType === 'step2')
+        return (
+            <>
+                <PassphraseHeading>
+                    <Translation id="TR_PASSPHRASE_WALLET_CONFIRMATION_STEP2_TITLE" />
+                </PassphraseHeading>
+                <PassphraseList>
+                    <PassphraseItem>
+                        <Icon icon="NEWSPAPER" size={16} />
+                        <Text>
+                            <Translation id="TR_PASSPHRASE_WALLET_CONFIRMATION_STEP2_ITEM1_DESCRIPTION" />
+                        </Text>
+                    </PassphraseItem>
+                    <PassphraseItem>
+                        <Icon icon="COPY" size={16} />
+                        <Text>
+                            <Translation id="TR_PASSPHRASE_WALLET_CONFIRMATION_STEP2_ITEM2_DESCRIPTION" />
+                        </Text>
+                    </PassphraseItem>
+                    <PassphraseItem>
+                        <Icon icon="HIDE" size={16} />
+                        <Text>
+                            <Translation id="TR_PASSPHRASE_WALLET_CONFIRMATION_STEP2_ITEM3_DESCRIPTION" />
+                        </Text>
+                    </PassphraseItem>
+                </PassphraseList>
+
+                <Warning>
+                    <Translation id="TR_PASSPHRASE_WALLET_CONFIRMATION_STEP2_WARNING" />
+                </Warning>
+                <Button
+                    isFullWidth
+                    variant="tertiary"
+                    onClick={() => {
+                        setContentType('step3');
+                    }}
+                    margin={{ top: 12 }}
+                >
+                    <Translation id="TR_PASSPHRASE_WALLET_CONFIRMATION_STEP2_BUTTON" />
+                </Button>
+            </>
+        );
+
+    if (contentType === 'step3')
+        return (
+            <>
+                <PassphraseHeading>
+                    <Translation id="TR_PASSPHRASE_WALLET_CONFIRMATION_STEP3_TITLE" />
+                </PassphraseHeading>
+                <Warning icon="INFO">
+                    <Translation id="TR_PASSPHRASE_WALLET_CONFIRMATION_STEP3_WARNING" />
+                </Warning>
+                <PassphraseTypeCard
+                    type="hidden"
+                    authConfirmation={authConfirmation}
+                    submitLabel={<Translation id="TR_CONFIRM_PASSPHRASE" />}
+                    offerPassphraseOnDevice={onDeviceOffer}
+                    onSubmit={onSubmit}
+                    singleColModal
+                    learnMoreTooltipOnClick={
+                        <OpenGuideFromTooltip
+                            dataTest="@tooltip/guideAnchor"
+                            id="/1_initialize-and-secure-your-trezor/6_passphrase.md"
+                        />
+                    }
+                />
+            </>
+        );
+
+    return null;
```

Comment:
If you add return type to this function what omits `null` and `undefined` you dont need this an as a bonus, if you add new step to the enum, typescript will tell you that it need to be handled here as well